pkg/azurefile-proxy/install-proxy.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/sh
+
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -xe
+
+if [ "${MIGRATE_K8S_REPO}" = "true" ]
+then
+  # https://kubernetes.io/blog/2023/08/15/pkgs-k8s-io-introduction/#how-to-migrate
+  echo "deb [signed-by=/etc/apt/keyrings/kubernetes-apt-keyring.gpg] https://pkgs.k8s.io/core:/stable:/v1.30/deb/ /" | tee /host/etc/apt/sources.list.d/kubernetes.list
+  $HOST_CMD curl -fsSL https://pkgs.k8s.io/core:/stable:/v1.30/deb/Release.key | $HOST_CMD gpg --dearmor --batch --yes -o /etc/apt/keyrings/kubernetes-apt-keyring.gpg
+fi
+
+
+if [ "${INSTALL_AZNFS_MOUNT}" = "true" ];then
+  echo "install aznfs-mount...."
+  # $HOST_CMD curl -fsSL https://github.com/Azure/AZNFS-mount/releases/download/0.1.526/aznfs_install.sh | $HOST_CMD bash
+  # shellcheck disable=SC1091
+  $HOST_CMD curl -sSL -O "https://packages.microsoft.com/config/$(. /host/etc/os-release && echo "$ID/$VERSION_ID")/packages-microsoft-prod.deb"
```

Comment:
we should install this https://github.com/Azure/AZNFS-mount/releases/download/0.1.526/aznfs_install.sh in agentbaker first

Reply:
I mean install `apt-get install -y --allow-downgrades aznfs=0.3.15` in AB first

Reply:
Yes, and we used `INSTALL_AZNFS_MOUNT` env  to control whether to install aznfs-mount by csi driver or not. We can skip installing when it has been installed in AB.